### PR TITLE
[DOCS] Update README

### PR DESCRIPTION
**Pull request by akashchi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,109 +2,146 @@
...
 
-[OpenVINO](https://github.com/openvinotoolkit/openvino)™ DL Workbench is a web-based
-graphical environment that enables users to visualize, fine-tune, and compare performance of deep
-learning models on various Intel® architecture configurations, such as CPU, Intel® Processor
-Graphics (GPU), Intel® Movidius™ Neural Compute Stick 2 (NCS 2), and Intel® Vision Accelerator
-Design with Intel® Movidius™ VPUs.
+## <a id="introduction">Introduction</a>
 
-The intuitive web-based interface of the DL Workbench allows you to easily use various sophisticated OpenVINO™ toolkit components:
+Deep Learning Workbench is a web-based graphical environment with a convenient user-friendly interface, and a wide range of customization options designed to make the development of deep learning models significantly easier.
```
Comment (tsavina) [inline README.md:16]:
```suggestion
Deep Learning Workbench is an official OpenVINO™ graphical user interface designed to make the production of pre-trained deep learning models significantly easier. 
```

File: README.md [hunk 2]
```diff
@@ -2,109 +2,146 @@
...
 
-* [Post-Training Optimization tool](https://docs.openvinotoolkit.org/latest/pot_README.html) to
-  calibrate a model and then execute it in the INT8 precision
+In the DL Workbench, you can use the following OpenVINO™ toolkit components:
 
-* [Accuracy Checker](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker.html) to
-  determine the accuracy of a model
+Component  |                 Description 
+|:------------------:|:------------------|
+| [Model Downloader and Model Converter](https://docs.openvinotoolkit.org/latest/omz_tools_downloader.html)| **Model Downloader** is a tool for getting access to the collection of high-quality pre-trained deep learning [public](https://docs.openvinotoolkit.org/latest/omz_models_group_public.html) and [Intel](https://docs.openvinotoolkit.org/latest/omz_models_group_intel.html) -trained models. The tool downloads model files from online sources and, if necessary, patches them with Model Optimizer. <br> **Model Converter** is a tool for converting the models stored in a format other than the Intermediate Representation (IR) into that format using Model Optimizer. |
```
Comment (tsavina) [inline README.md:30]:
```suggestion
| [Open Model Zoo](https://docs.openvinotoolkit.org/latest/omz_tools_downloader.html)| Get access to the collection of high-quality pre-trained deep learning [public](https://docs.openvinotoolkit.org/latest/omz_models_group_public.html) and [Intel-trained](https://docs.openvinotoolkit.org/latest/omz_models_group_intel.html) models trained to resolve a variety of different tasks.  |
```

File: README.md [hunk 3]
```diff
@@ -2,109 +2,146 @@
...
+| [Model Optimizer](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) |**Model Optimizer** imports, converts, and optimizes models that were trained in certain frameworks to the IR format used in OpenVINO tools. <br>Supported frameworks include TensorFlow\*, Caffe\*, Kaldi*\*, MXNet\*, and ONNX\*.
```
Comment (tsavina) [inline README.md:31]:
```suggestion
| [Model Optimizer](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) |Optimize and transform models trained in supported frameworks to the IR format. <br>Supported frameworks include TensorFlow\*, Caffe\*, Kaldi\*, MXNet\*, and ONNX\* format.  
```

File: README.md [hunk 4]
```diff
@@ -2,109 +2,146 @@
...
+| [Model Optimizer](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) |**Model Optimizer** imports, converts, and optimizes models that were trained in certain frameworks to the IR format used in OpenVINO tools. <br>Supported frameworks include TensorFlow\*, Caffe\*, Kaldi*\*, MXNet\*, and ONNX\*.  
+| [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html)| **Benchmark Application** allows you to estimate deep learning inference performance on supported devices for synchronous and asynchronous modes.
```
Comment (tsavina) [inline README.md:32]:
```suggestion
| [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html)| Estimate deep learning model inference performance on supported devices.
```

File: README.md [hunk 5]
```diff
@@ -2,109 +2,146 @@
...
+| [Model Optimizer](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) |**Model Optimizer** imports, converts, and optimizes models that were trained in certain frameworks to the IR format used in OpenVINO tools. <br>Supported frameworks include TensorFlow\*, Caffe\*, Kaldi*\*, MXNet\*, and ONNX\*.  
+| [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html)| **Benchmark Application** allows you to estimate deep learning inference performance on supported devices for synchronous and asynchronous modes.   
+| [Accuracy Checker](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker.html) |**Accuracy Checker**  is a deep learning accuracy validation tool that allows you to evaluate accuracy on the given dataset by collecting one or several metric values.
```
Comment (tsavina) [inline README.md:33]:
```suggestion
| [Accuracy Checker](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker.html) |Evaluate the accuracy of a model by collecting one or several metric values. 
```

File: README.md [hunk 6]
```diff
@@ -2,109 +2,146 @@
...
+| [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html)| **Benchmark Application** allows you to estimate deep learning inference performance on supported devices for synchronous and asynchronous modes.   
+| [Accuracy Checker](https://docs.openvinotoolkit.org/latest/omz_tools_accuracy_checker.html) |**Accuracy Checker**  is a deep learning accuracy validation tool that allows you to evaluate accuracy on the given dataset by collecting one or several metric values. 
+| [Post-Training Optimization Tool](https://docs.openvinotoolkit.org/latest/pot_README.html)|**Post-Training Optimization Tool** allows you to optimize trained models with advanced capabilities, such as quantization and low-precision optimizations, without the need to retrain or fine-tune models.                               |
```
Comment (tsavina) [inline README.md:34]:
```suggestion
| [Post-Training Optimization Tool](https://docs.openvinotoolkit.org/latest/pot_README.html)|Optimize pre-trained models with lowering the precision of a model from floating-point precision(FP32 or FP16) to integer precision (INT8), without the need to retrain or fine-tune models.                              |
```

File: README.md [hunk 7]
```diff
@@ -2,109 +2,146 @@
...
+To successfully run the DL Workbench with Python Starter, install Python 3.6 or higher.
+
+Prerequisite | Linux* | Windows* | macOS*
+:----- | :----- |:----- |:-----
+Operating system|Ubuntu\* 18.04|Windows\* 10 | macOS\* 10.15 Catalina
+Available RAM space| 8 GB\** | 8 GB\** | 8 GB\**
+Available storage space| 10 GB + space for imported artifacts| 10 GB + space for imported artifacts| 10 GB + space for imported artifacts
+Docker\*| Docker CE 18.06.1 | Docker Desktop 2.3.0.3|Docker CE 18.06.1
+
+Windows*, Linux* and MacOS* support CPU targets. GPU, Intel® Neural Compute Stick 2 and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs are supported only for Linux*.
```
Comment (tsavina) [inline README.md:49]:
```suggestion
Windows\*, Linux\* and MacOS\* support CPU targets. GPU, Intel® Neural Compute Stick 2 and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs are supported only for Linux\*.
```

File: README.md [hunk 8]
```diff
@@ -2,109 +2,146 @@
...
-   curl -o start_workbench.sh https://raw.githubusercontent.com/openvinotoolkit/workbench_aux/master/start_workbench.sh
-   ```
+To start the latest available version of the DL Workbench, execute the following command:
 
-2. Open a terminal in the directory with the downloaded script and run the following command to make
-   the file executable:
-```bash
-chmod +x start_workbench.sh
+```
+openvino-workbench --image openvino/workbench:2021.3
```
Comment (tsavina) [inline README.md:113]: 2021.3 or 2021.4?

Comment (akashchi) [reply README.md:113]: It will be updated to `21.4` once it is released.

**Review by tsavina — APPROVED** [review]

**Review by artyomtugaryov — APPROVED** [review]

**Pull request merged**